pennylane_qiskit/converter.py

```diff
@@ -91,15 +154,38 @@ def _check_circuit_and_bind_parameters(
             "The circuit {} is not a valid Qiskit QuantumCircuit.".format(quantum_circuit)
         )
 
+    # confirm parameter names are valid for conversion to PennyLane
+    for name in ["wires", "params"]:
+        if name in [p.name for p in quantum_circuit.parameters]:
+            raise RuntimeError(
+                f"Cannot interpret QuantumCircuit with parameter '{name}' as a PennyLane "
+                f"quantum function, as this argument is reserved"
+            )
+
     if params is None:
+        if quantum_circuit.parameters:
+            raise TypeError(
+                f"Missing required argument{'s' if len(quantum_circuit.parameters) > 1 else ''} "
```

Comment:
Maybe we can store `{'s' if len(quantum_circuit.parameters) > 1 else ''}` above so that we don't have to write it twice or simply use `argument(s)` lol.